lib/app/modules/core/capability_statement_sequence.rb

```diff
@@ -71,9 +71,23 @@ class CapabilityStatementSequence < SequenceBase
         end
       end
 
-      test 'FHIR server supports the conformance interaction that defines how it supports resources' do
+      test 'FHIR version of the server matches the FHIR version expected by tests' do
         metadata do
           id '02'
+          link 'http://www.hl7.org/fhir/directory.cfml'
+          desc %(
+            Checks that the FHIR version of the server matches the FHIR version expected by the tests.
+            This test will inspect the CapabilityStatement returned by the server to verify the FHIR version of the server.
+          )
+        end
+
```

Comment:
Technically @instance.fhir_version can be `nil`, which means that the tests you are running are version independant.  This is unusual, but is the case for our SMART on FHIR launch tests technically.  Just add a line like this to automatically have them pass in this case:
```
pass 'Tests are not version dependent' if @instance.fhir_version.blank?
```

Having a test `pass` doesn't make a ton of sense, but it is the closest status we have up to this point.  A ticket for you to address in the near future would be to add an `omit` status, which would be perfect for these types of cases.

Reply:
Pushed new version with this fix